docs/docs/rasa-pro-setup.mdx

```diff
@@ -0,0 +1,120 @@
+---
+id: rasa-pro-setup
+sidebar_label: Rasa Pro Setup
+title: Rasa Pro Setup
+description: Installation of Rasa Pro
+abstract: Install Rasa Pro in your production environment to create an enterprise ready Rasa deployment.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Rasa Pro is compromised of two packages. The packages need to be installed
+separately and in different environments:
+
+- **Rasa Plus**, a drop in replacement for Rasa Open Source for an
+  enterprise deployment. Rasa Plus should be used to deploy your assistant
+  into production.
+- **Rasa Pro Services**, flexible infrastructure and APIs on top of Rasa
+  Open Source. Rasa Pro Services should be deployed alongside, but separately
+  from your production assistant. Rasa Pro Services will connect to your
+  production assistant using Kafka.
+
+## Rasa Plus
+
+:::warning
+I think this content is written currently in the rasa-plus repo.
+:::
+
+## Rasa Pro Services
+
+Installation of the data pipeline requires the deployment of a docker
+container. The container will receive data from a Kafka instance and
```

Comment:
Are we conflating installation/configuration of analytics pipeline with installation/configuration of Rasa Pro services?